### DISCUSSION TITLE
Ability to create presets for the settings menu

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20443] [Ideas] [answered]
It would be nice to be able to switch presets for different playstyles (ex. different area for left and right handed aiming, different keys for touchscreen and mouse, different key preferences for mania, different visual settings for different skins, etc.) and of course for different players on the same computer without having to spend a long time switching the settings up every time.
Presets would save everything in the settings, including skin, offset, and keybinds.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/18080

Please search before posting. I searches "settings presets" and found this easily.
